tools/test-js-with-node

```diff
@@ -201,6 +202,7 @@ EXEMPT_FILES = make_set(
         "web/src/sent_messages.ts",
         "web/src/sentry.ts",
         "web/src/server_events.js",
+        "web/src/server_events_dispatch.js",
```

Comment:
Why is 100% test coverage removed on these files?

Reply:
Added some checks for 100% coverage.